.github/workflows/build_and_test.yml

```diff
@@ -93,41 +83,56 @@ jobs:
     - name: Check labeler configuration
       run: scripts/check-labeler.pl .github/labeler.yml
 
-    - name: Unit tests
-      env:
-        KONG_TEST_PG_DATABASE: kong
-        KONG_TEST_PG_USER: kong
-      run: |
-          source ${{ env.BUILD_ROOT }}/kong-dev-venv.sh
-          TEST_CMD="bin/busted -v -o htest spec/01-unit"
-          if [[ $KONG_TEST_COVERAGE = true ]]; then
-            TEST_CMD="$TEST_CMD --coverage"
-          fi
-          $TEST_CMD
+  schedule:
+    name: Schedule busted tests to run
+    runs-on: ubuntu-22.04
+    needs: build
 
-    - name: Archive coverage stats file
+    env:
+      WORKFLOW_ID: ${{ github.run_id }}
+
+    outputs:
+      runners: ${{ steps.generate-runner-array.outputs.RUNNERS }}
+
+    steps:
+    - name: Checkout source code
+      uses: actions/checkout@v4
+
+    - name: Download runtimes file
+      uses: Kong/gh-storage/download@main
+      with:
+        repo-path: Kong/gateway-action-storage/main/.ci/runtimes.json
+
+    - name: Schedule tests
+      uses: Kong/gateway-test-scheduler/schedule@main
+      with:
+        test-suites-file: .ci/test_suites.json
+        test-file-runtime-file: .ci/runtimes.json
+        output-prefix: test-chunk.
+        runner-count: ${{ env.RUNNER_COUNT }}
+
+    - name: Upload schedule files
       uses: actions/upload-artifact@v3
-      if: ${{ always() && (inputs.coverage == true || github.event_name == 'schedule') }}
+      continue-on-error: true
       with:
-        name: luacov-stats-out-${{ github.job }}-${{ github.run_id }}
-        retention-days: 1
-        path: |
-          luacov.stats.out
+        name: schedule-test-files
+        path: test-chunk.*
+        retention-days: 7
```

Comment:
would be nice if we can `cat` the content of two schedule files into `$GITHUB_STEP_SUMMARY` so we can view it without downloading the artifact

Reply:
I'll add a report to the schedule action that makes it easy to find which runner runs what.

Reply:
Please have a look at the output of the "Schedule Tests" step in https://github.com/Kong/kong/actions/runs/7207036134/job/19653095500 - Is this what you've asked for?  I decided to not re-sort the output so that one can see the order in which tests are run.  To find a particular test file, one needs to search.  If a list sorted by test filename and suite mapping to runner would be useful, I can also include that.